TUTORIAL.md

```diff
@@ -211,11 +228,11 @@ node {
 }
 ```
 
-Properties of this variable will be environment variables on the current node.
-You can also override certain environment variables, and the overrides will be seen by subsequent `sh` steps (or anything else that pays attention to environment variables).
-This is convenient because now we can run `mvn` without a fully-qualified path.
+* Properties of this variable are environment variables on the current node.
+* You can also override certain environment variables, and the overrides will be seen by subsequent `sh` steps (or anything else that pays attention to environment variables).
+* You can run `mvn` without a fully-qualified path.
```

Comment:
Split into a bullet list this no longer makes any sense. The original paragraph meant that _if_ you override `PATH`, _then_ you can run `mvn` without a fully-qualified path.